src/languageSupport/languages/flux/lsp/connection.ts

```diff
@@ -248,7 +248,7 @@ class LspConnectionManager {
       },
     ]
 
-    const removeAllStyles = !compositionBlock && schema.composition.diverged
+    const removeAllStyles = schema.composition.diverged
```

Comment:
I think it should technically remove all styles when either: (1) there is no compositionBlock in the editor, or (2) it's diverged.

`!compositionBlock || schema.composition.diverged`